content/src/content/jcr_root/apps/core/wcm/components/form/options/v2/options/amp.html

```diff
@@ -0,0 +1,39 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.options="com.adobe.cq.wcm.core.components.models.form.Options"
```

Comment:
@gaetanmarmasse - I only see the following diff from the base markup, basically the re-ordering of one attribute. Do we need this amp-specific rendering for the form options component?
<img width="1004" alt="Screenshot 2020-01-29 at 12 11 08" src="https://user-images.githubusercontent.com/25616660/73352066-9c1b9a80-4290-11ea-8238-36c4c680d356.png">

Reply:
Nope, we don't, good catch. Created https://github.com/Bounteous-Inc/aem-core-wcm-components/pull/56 that simply removes the file.